app/models/concerns/triple_eye_effable/resourceable.rb

```diff
@@ -13,6 +13,7 @@ module Resourceable
       delegate :content_url, to: :resource_description, allow_nil: true
       delegate :content_download_url, to: :resource_description, allow_nil: true
       delegate :content_iiif_url, to: :resource_description, allow_nil: true
+      delegate :content_info_url, to: :resource_description, allow_nil: true
```

Comment:
I think we'll need to add this attribute to the `resource_descriptions` model.

Reply:
Isn't that this? Or am I missing a step? 

https://github.com/performant-software/triple-eye-effable/blob/7f91a0752b26e1f524ec491d6ce25ce4bdd52349/app/models/triple_eye_effable/resource_description.rb#L10

Reply:
Ah yes, I missed that somehow. 👍